drivers/sensor/sht21/Kconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020 Philémon Jaermann
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SHT21
+	bool "SHT21 Humidity sensor"
+	depends on I2C && HAS_DTS_I2C
+	help
+	  Enable driver for SHT21 I2C-based Humidity sensor.
+	  This driver also support the HTU21D and the SI7021
+	  clones.
+
+if SHT21
+
+choice SHT21_MEAS_RES
+	prompt "Measurement resolution"
+	default SHT21_MEAS_RES_RH_12_BITS
+	help
+	  Specify the desired measurement resolution.
+
+config SHT21_MEAS_RES_RH_8_BITS
```

Comment:
As a maintainer looking at the implementation: it would be nice if the symbol name encoded both the RH and T resolutions, because they jump around.  Perhaps 12RH_14T (0), 8RH_12T (1), 10RH_13T (2), and 11RT_11T (3).